tox.ini

```diff
@@ -66,3 +66,18 @@ deps={[testenv:optional]deps}
 envdir={[testenv:optional]envdir}
 recreate={[testenv:fast]recreate}
 setenv={[testenv:fast]setenv}
+
```

Comment:
I think that makes sense. The reason I hesitated is because if/when we want to move to using Tox in its more typical usage pattern (e.g. with Django 1.8/1.11 or Python 2/3) then these kind of one-off environments will be a little more tricky to include.